src/ecs/resources/mod.rs

```diff
@@ -11,6 +11,6 @@ mod broadcaster;
 
 pub use self::broadcaster::Broadcaster;
 pub use self::camera::{Camera, Projection};
-pub use self::input::{Axis, Button, InputHandler, PressedButtons, PressedKeys, PressedMouseButtons};
+pub use self::input::{Axis, Button, Buttons, KeyCodes, MouseButtons, InputHandler};
```

Comment:
Alphabetical ordering together with the maximum line width is super annoying. Gotta redo quite a few lines just to make a simple change :(